website/js/submission.js

```diff
@@ -1,23 +1,32 @@
+// Prevents the standard form submission from POSTing and allows the JS below to work
+// TODO: CB Is this still required? I spot its duplicated elsewhere!
 $("#tagForm").submit(function(e){
     e.preventDefault();
-    alert("Hello"); 
 })

```

Comment:
You can probably remove this, as you say there is a duplicate further down in the page and that should be able to handle the default processes of the submit.  (as long as the tagForm submit box is there on document ready) which it should be.

Reply:
Technically this update does remove it ;)